dali/pipeline/data/tensor.h

```diff
@@ -183,7 +183,7 @@ class Tensor : public Buffer<Backend> {
    * @brief Tensor is always backed by contiguous buffer
    *        Cannot be set to noncontiguous
    */
-  void SetContiguous(bool contiguous) {
+  void SetBatchState(bool contiguous) {
```

Comment:
Not quite sure what benefit we have from this method, since we have `enum BatchState` version.

Reply:
Never mind, I see it's on TODO list.

Reply:
Done